x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/WaitForNoFollowersStepTests.java

```diff
@@ -64,7 +68,7 @@ protected WaitForNoFollowersStep copyInstance(WaitForNoFollowersStep instance) {
         return new WaitForNoFollowersStep(instance.getKey(), instance.getNextStepKey(), instance.getClient());
     }
 
-    public void testConditionMet() {
+    public void testConditionMetWhenEnable() {
```

Comment:
Shall we name this `testConditionMetWhenCCREnabled` ?

Reply:
++